cmd/brimstone/main.go

```diff
@@ -159,20 +158,11 @@ func main() {
 		e.Logger.Fatalf("failed to initialize database: %s", initdbErr)
 	}
 
-	e.Logger.Fatal(e.Start(net.JoinHostPort("0.0.0.0", *port)))
-}
-
-func GetAPIKey() (string, bool) {
-	return os.LookupEnv(API_KEY_VARNAME)
-}
-func GetGGAPIToken() (string, bool) {
-	return os.LookupEnv(GG_API_TOKEN_VARNAME)
-}
-func GetGGWebhookToken() (string, bool) {
-	return os.LookupEnv(GG_WEBHOOK_TOKEN_VARNAME)
+	e.Logger.Fatal(e.Start(net.JoinHostPort("0.0.0.0", strconv.Itoa(int(cfg.Port)))))
```

Comment:
Can this be simplified by declaring the Port as an "int" in the struct definition?